node/resolvers/checkout/types.ts

```diff
@@ -18,7 +18,7 @@ export interface AssemblyOption {
     minQuantity: number
     maxQuantity: number
     items: CompositionItem[]
-  }
+  } | null
```

Comment:
What is this pipe doing here?

Reply:
It means that this property is either `A | null`